### DISCUSSION TITLE
How to troubleshoot this

### DISCUSSION THREAD

**Discussion by RobinSchell** [apohl79/audiogridder#383] [Help] [answered]
I have AG running, plugins installed on the server machine, everything opens up and runs as it should. But I am getting major glitching and crashing even with only one reverb plugin via AG.  I suspect it's a network issue but I don't know where to start.

the setup is:
Main DAW is Reason 10, win 7 pro,  3rd gen i5, 16GB RAM

Server is 4th gen i5, win 10, 8GB RAM 1Gb Lan using a D-link switch

On another forum I saw someone mentioning winVNC , should I try that?

**Comment by apohl79** [comment]
If your server is headless, try VNC. RDP does not work most of the time. You need an active UI login on windows, and VNC keeps it open, even when you close the connection.

You can check the logfiles to see if you get any clues from them (https://audiogridder.com/bug-report-diagnostics-locations/).

If that does not help, follow the bug reporting guide (https://audiogridder.com/bug-reports/) and reproduce the problem you have, than attach the files here or use wetransfer (https://audiogridder.com/submitting-large-files/).

**Reply by RobinSchell** [reply]
My server is not headless, it has it's own monitor, keyboard and mouse. (unless 'headless' means something else :-o   - all this network stuff is, as they say, Greek to me)

**Comment by RobinSchell** [comment, marked as answer]
The issue appears to be solved by increasing the DAW buffer (1024 samples).

**Reply by apohl79** [reply]
thanks for sharing
